Purchase.java

```diff
@@ -0,0 +1,77 @@
+package by.epam.lab;
+
+public class Purchase implements Comparable<Purchase> {
+
+    private final String PRODUCT_NAME = "milk";
+    private final int PRICE = 100;
+    private int number;
+    private int percent;
+    public enum WeekDay {
+        SUNDAY,
+        MONDAY,
+        TUESDAY,
+        WEDNESDAY,
+        THURSDAY,
+        FRIDAY,
+        SATURDAY
+    }
+    public WeekDay weekDay;
+
+    public Purchase() {
+    }
+
+    public Purchase(int number, int percent, WeekDay weekDay) {
+        this.number = number;
+        this.percent = percent;
+        this.weekDay = weekDay;
+    }
+
+    public String getPRODUCT_NAME() {
+        return PRODUCT_NAME;

```

Comment:
You don't need getters and setters for static finals (constants). If they are used somewhere else, they can be just public, and access to them looks like Class.CONST (for ex. Purchase.PRICE)